vanity-pet.lic

```diff
@@ -11,23 +11,36 @@ class VanityPet
   def initialize
     arg_definitions = [
       [
-        { name: 'pet_action', regex: /(stow|drop)/, optional: false, description: 'STOW or DROP to either stow your pet in their container or drop them to the ground.' },
-        { name: 'pet_name', regex: /(\w+)/, optional: false, description: 'Name of your pet.' },
-        { name: 'pet_container', regex: /(\w+)/, optional: false, description: 'Container for your pet.' },
-        { name: 'debug', regex: /debug/i, optional: true, description: 'Enable debug output' },
+        { name: 'pet_action', regex: /(stow|drop)/, optional: true, description: 'STOW or DROP to either stow your pet in their container or drop them to the ground.' },
+        { name: 'pet_name', regex: /^(?!debug)\w+$/, optional: true, description: 'Name of your pet.' },
+        { name: 'pet_container', regex: /^(?!debug)\w+$/, optional: true, description: 'Container for your pet.' },
+        { name: 'debug', regex: /^debug$/i, optional: true, description: 'Enable debug output' },
```

Comment:
The negative-lookbehind ensured that if the argument `debug` is provided then it won't match as the pet name nor the pet container.

This allows calling the script `;vanity-pet debug` and the name/container defaulting to the yaml settings. 🎉